docs/guide/net.rst

```diff
@@ -70,19 +71,115 @@ This will return a list-like object which contains one or more filepaths to the
 Downloading FITS files with Globus
 ----------------------------------
 
-As is detailed in the next section :ref:`loadinglevel1data` these can be passed to `dkist.Dataset` to open the file.
-For example, to load only the first ASDF file we would do:
+As mentioned at the top of the page the DKIST level one data are only available to download using the `Globus <https://www.globus.org/data-transfer>`__ file transfer system.
+The `dkist` package provides helpers to orchestrate the transfer of files with Globus.
+The objective of these helpers is to provide the tools needed to quickly access some or all of a dataset, and to script the transfer of any data.
+
+The `dkist` package provides two different interfaces for starting a Globus transfer.
+One is `~dkist.net.transfer_whole_dataset` which will stage the whole dataset directory for download.
+The other is the `~dkist.io.FileManager.download` method of ``Dataset.files``, this method allows you to only download some FITS files based on the slicing of a dataset object.
+
+
+Downloading Files from a ``Dataset``
+####################################
+
+The most flexible way to download FITS files is first having loaded the dataset from an ASDF file.
```

Comment:
"...first having loaded the dataset..." is awkward; I'm not even sure what it's trying to say. Would "...to first load the dataset from an ASDF file." correct?